vendor/profile.ps1

```diff
@@ -74,7 +74,7 @@ $env:Path = "$Env:CMDER_ROOT\bin;$env:Path;$Env:CMDER_ROOT"
 $CmderUserProfilePath = Join-Path $env:CMDER_ROOT "config/user-profile.ps1"
 if(Test-Path $CmderUserProfilePath) {
     # Create this file and place your own command in there.
-    Invoke-Expression $CmderUserProfilePath
+    & $CmderUserProfilePath
```

Comment:
I looked it up, the difference between dot sourcing and the ampersand is that dot sourcing gives the command prompt access to the variables defined in the script after execution, with an ampersand you can't access them from the command prompt.